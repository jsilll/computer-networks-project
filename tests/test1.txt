reg 12345 password
reg 12345 password
unr 12345 passwora
unr 12345 password
unr 54321 password
login 12345 password 
reg 12345 password
login 12345 passwora
login 54321 passwora
login 12345 password
login 12345 password
logout
logout