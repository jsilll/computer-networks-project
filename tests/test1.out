RRG OK
RRG DUP
RUN NOK
RUN OK
RUN NOK
RLO NOK
RRG OK
RLO NOK
RLO NOK
RLO OK
RLO OK
ROU OK
User needs to be logged in